pkgs/applications/audio/tonelib-gfx/default.nix

```
@@ -0,0 +1,56 @@
+{ stdenv
+, dpkg
+, autoPatchelfHook
+, fetchurl
+, gtk3
+, glib
+, desktop-file-utils
+, alsaLib
+, libjack2
+, harfbuzz
+, fribidi
+, pango
+, freetype
+}:
+
+stdenv.mkDerivation
+{
+  name = "tonelib-gfx";
+  src = fetchurl
+    {
+      url = "https://www.tonelib.net/download/ToneLib-GFX-amd64.deb";
+      sha256 = "sha256-wdX3SQSr0IZHsTUl+1Y0iETme3gTyryexhZ/9XHkGeo=";
+    };
```

```suggestion
stdenv.mkDerivation {
  pname = "tonelib-gfx";
  version = "";

  src = fetchurl {
    url = "https://www.tonelib.net/download/ToneLib-GFX-amd64.deb";
    sha256 = "sha256-wdX3SQSr0IZHsTUl+1Y0iETme3gTyryexhZ/9XHkGeo=";
  };

```
What version is that package?
Also please upload it to web.archive.org to prevent hash changes by updates.

---

I'll figure it out well now, add a maintainer. But I'm not on the maintainer list yet. Will I be added to the list before adding the package?
> maintainers = with maintainers; [dan4ik605743];
To avoid the error dan4ik605743 not found